CI/vagrant/Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+$script = <<SCRIPT
+# kill apt that is started on boot time
+sudo systemctl stop apt-daily.service
+sudo systemctl stop apt-daily.timer
+sudo systemctl disable apt-daily.service
+sudo systemctl disable apt-daily.timer
+
+sudo systemctl kill --kill-who=all apt-daily.service                                                    
+
+# wait until `apt-get updated` has been killed
+while ! (sudo systemctl list-units --all apt-daily.service | fgrep -q dead)                             
+do
+  sleep 1;
+done
+
+# wait unitl other software managers are done
+while sudo fuser /var/lib/dpkg/lock >/dev/null 2>&1; do sleep 1; done                                   
+
+# download and install iovnet and all the dependencies                                                  
+set -e
+echo Downloading iovnet
+git clone git@github.com:netgroup-polito/iovnet.git
```

Comment:
Why are you downloading code from "netgroup-polito"? Shouldn't it be from polycube-network?